frontend/src/contexts/round.tsx

```diff
@@ -38,40 +44,11 @@ export type RoundContextType = {
   setWinner: (winningPlayerId: string, winningPunchlines: string[]) => void;
 };
 
-const RoundContext = React.createContext<RoundContextType>({
-  // round:number
-  roundNumber: 0,
-  setRoundNumber: () => null,
-
-  // round:setup
-  setup: {
-    setup: "",
-    type: "PICK_ONE",
-  },
-  setSetup: () => null,
-
-  // round:increment-players-chosen
-  numPlayersChosen: 0,
-  incrementPlayersChosen: () => null,
-
-  // round:chosen-punchlines
-  punchlinesChosen: [],
-  setPunchlinesChosen: () => null,
-  markPunchlineRead: () => null,
-
-  // round:host-view
-  hostViewIndex: 0,
-  setHostViewIndex: () => null,
-
-  // round:winner
-  winner: {
-    winningPlayerId: "",
-    winningPunchlines: [],
-  },
-  setWinner: () => null,
-});
+const RoundContext = React.createContext<RoundContextType | undefined>(
+  undefined
+);
 
-const RoundContextProvider = ({ children }: { children: React.ReactNode }) => {
+export const RoundProvider = ({ children }: PropsWithChildren<unknown>) => {
```

Comment:
I notice L14/L20 has the export removed - is this just so the provider can be directly accessed rather than the type? Not very familiar with context providers so not sure of the rationale/breadth of this change

Reply:
In short it just wasn't necessary to include the export for context type anymore as the only way to access the context is through the `useRound()` hook which returns the context type - should be no need to reference the context type outside of this file anymore.